src/System.Private.CoreLib/shared/System/Threading/Tasks/Task.cs

```diff
@@ -3296,11 +3296,10 @@ internal void FinishContinuations()
         {
             Debug.Assert(continuationObject != null);
 
-            TplEventSource log = TplEventSource.Log;
-            bool TplEventSourceLoggingEnabled = log.IsEnabled();
-            if (TplEventSourceLoggingEnabled)
+            TplEventSource? log = TplEventSource.Log;
```

Comment:
>TplEventSource.Log [](start = 34, length = 18)

TplEventSource.Log is always not null?

Reply:
Yes

Reply:
wouldn't the next check will wrong then?

if (!log.IsEnabled())

as log can be null?


---
In reply to: [295891558](https://github.com/dotnet/coreclr/pull/25280#discussion_r295891558) [](ancestors = 295891558)

Reply:
Log is always not null.

Reply:
ignore my last comment. I just confused myself :-)

Reply:
Ok :)